src/without.js

```diff
@@ -0,0 +1,25 @@
+var _contains = require('./internal/_contains');
+var flip = require('./flip');
+var identity = require('./identity');
+var reject = require('./reject');
+var useWith = require('./useWith');
+
+/**
+ * Returns a new list without values in the first argument.
+ * `R.equals` is used to determine equality.
+ *
+ * Acts as a transducer if a transformer is given in list position.
+ * @see R.transduce
```

Comment:
Please move this line to between the `@return` and `@example` annotations (see #1458).